downscale/estimate_forcing_regression.f90

```diff
@@ -1,7 +1,7 @@
 ! AWW-2016Jan, modifications to handle time subsetting and reduce mem alloc, and clean up
 !   renamed from estimate_precip; add also 'directory' var, changed some var names
 
-subroutine estimate_forcing_regression (nPredict, gen_sta_weights, sta_weight_name, nwp_input_list, &
+subroutine estimate_forcing_regression (sta_limit, nPredict, gen_sta_weights, sta_weight_name, nwp_input_list, &
```

Comment:
it's good to use descriptive variable names, eg n_training_stns or training_stn_sample_size ... to this point also changing nPredict to n_predictors, though I have a question about whether than could be allocated on based on inputs rather than specified in a config file.

Reply:
nPredict could be set based on the number of inputs the user specifies rather than having it in the config file.  We should do that in another request?